src/gpodder/model.py

```diff
@@ -145,6 +145,14 @@ def from_podcastparser_entry(cls, entry, channel):
         episode.title = entry['title']
         episode.link = entry['link']
         episode.description = entry['description']
+        if entry.get('description_html'):
+            episode.description_html = entry['description_html']
+        # XXX: That's not a very well-informed heuristic to check
+        # if the description already contains HTML. Better ideas?
+        elif '<' in entry['description']:
+            episode.description_html = entry['description']
+        else:
+            episode.description_html = entry['description'].replace('\n', '<br>')
```

Comment:
Do I understand correctly that this will result in the description for every episode being stored twice?  Once under `description` and once under `description_html`?